### PR TITLE
[agent] change the default journald directory

### PR DESCRIPTION
**Pull request by atoulme**
**Description:**
Change the default journald directory to match the default of the receiver:
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/journaldreceiver

### CODE REVIEW

**Review by jvoravong — APPROVED** [review]

File: helm-charts/splunk-otel-collector/values.yaml [hunk 1]
```diff
@@ -599,8 +599,8 @@ logsCollection:
 # Configuration for collecting journald logs using otel collector
   journald:
     enabled: false
-    # Please update directory path for journald if it's different from below default value "/var/log/journal"
-    directory: /var/log/journal
+    # Please update directory path for journald if it's different from below default value "/run/log/journal"
+    directory: /run/log/journal
```
Comment (rmfitzpatrick) [inline helm-charts/splunk-otel-collector/values.yaml:603]: Why would we need to specify a value to begin with?

Comment (atoulme) [reply helm-charts/splunk-otel-collector/values.yaml:603]: oh true, the default value can be an empty string.

Comment (rmfitzpatrick) [reply helm-charts/splunk-otel-collector/values.yaml]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0d4683eb03d3f1aebb427fa094e0baff638deab6/pkg/stanza/operator/input/journald/journald.go#L101 empty string would still cause setting the `--directory` option so I think leaving the field commented would be preferable?

Comment (atoulme) [reply helm-charts/splunk-otel-collector/values.yaml]: understood, I'll revert and use the `/run/log/journal` then.

**Pull request merged**